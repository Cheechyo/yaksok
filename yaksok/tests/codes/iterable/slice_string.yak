패턴 = "강강약강강강약강중약"
패턴[1:3] 보여줘
패턴[4:7] 보여줘
패턴[8:10] 보여줘
