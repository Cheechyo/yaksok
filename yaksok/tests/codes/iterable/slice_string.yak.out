강강약
강강강약
강중약
